### PR TITLE
Adding FIRMWARE_VERSION 4.4 to some OTHER tunes

### PR DESCRIPTION
**Pull request by limonspb**
Remaining to check that im not sure (from OTHER):
- reset_launch_mode.txt
- reset_names.txt
- reset_servo.txt
- @TehllamaFPV  OTHER presets

### CODE REVIEW

**Review by ctzsnooze — DISMISSED** [review]

**Review by haslinghuis — APPROVED** [review]

**Comment by ctzsnooze** [comment]
PS @limonspb one thing to note is that in 4.4 there are only 4 rate profiles and 4 PID profiles.  Some Presets had settings for PID profiles 5 and 6, which will cause errors in 4.4.

File: presets/4.3/other/Street_League_motor_outputs.txt [hunk 1]
```diff
@@ -1,141 +1,136 @@
...
+            set throttle_limit_percent = 75
+        rateprofile 3
+            set throttle_limit_type = scale
+            set throttle_limit_percent = 75
+        rateprofile 4
+            set throttle_limit_type = scale
+            set throttle_limit_percent = 75
+        rateprofile 5
+            set throttle_limit_type = scale
+            set throttle_limit_percent = 75
```
Comment (haslinghuis) [inline presets/4.3/other/Street_League_motor_outputs.txt:46]: Please remove

**Review by haslinghuis — CHANGES_REQUESTED** [review]

**Comment by TehllamaFPV** [comment]
It was the rateprofiles.
I'm also encountering silly problems where the addition of the fourth profile is missing in some 'back to default' behaviors.  I need to go back through there - so far it's been a nonissue because it's all been fresh flashes for me, but that's the kind of unit testing that I'm bad about doing

**Comment by limonspb** [comment]
> PS @limonspb one thing to note is that in 4.4 there are only 4 rate profiles and 4 PID profiles. Some Presets had settings for PID profiles 5 and 6, which will cause errors in 4.4.

good catch!

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**